web/src/components/modals/add-view-options.tsx

```diff
@@ -0,0 +1,184 @@
+import { FC, useState } from 'react';
+import { Modal, Tab, Tabs } from 'react-bootstrap';
+import ReactSelect from 'react-select';
+import { Controller, FieldErrors, useForm } from 'react-hook-form';
+import { ErrorMessage } from '@hookform/error-message';
+
+
+import { useViewMutations } from '../../hooks/mutations/useViewMutations';
+import { useProjectPage } from '../../contexts/project-page-context';
+import { CreateProjectViewRequest, addProjectView, deleteProjectView } from '../../api/project';
+import { useProjectViews } from '../../hooks/queries/useProjectViews';
+import { useProjectSelectedView } from '../../hooks/stores/useProjectSelectedViewStore';
+
+interface Props {
```

Comment:
I used to use `interface` everywhere, and I don't know why I think I just saw it used. But I think we should be using `type` ... they are nearly interchangeable but I've been trying to replace `interface`s where I see them with `type`

https://www.reddit.com/r/typescript/comments/16vlmmx/interface_vs_type/

So this just becomes `type Props = { }`